docusaurus.config.js

```diff
@@ -85,7 +85,7 @@ module.exports = {
               className: "footer-slack-link",
             },
             {
-              label: "Twitter",
+              label: "𝕏",
               href: "https://twitter.com/pecanproject",
```

Comment:
we should change the url in that case as well, and ideally the footer-twitter-link id

Reply:
footer-x-link?